students/kula_marta/lesson_01_basics/lesson1_Area_of_right_angled_triangle.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+
+print("lekcja 2")
+b = int(input("set base: "))
+h = int(input("set height: "))
+p = 0.5 * b * h
+print("area = %d" % p)
```

Comment:
This answer doesn't pass Snakify tests

Please avoid adding not required print statement like 'print("lekcja 2")' and 'print("area = %d"..'

Correct example:
b = int(input("set base: "))
h = int(input("set height: "))
p = 0.5 * b * h
print(p)